packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -47,18 +47,22 @@ export class FocusTrapZone extends React.Component<IFocusTrapZoneProps, {}> impl
   public componentDidUpdate(prevProps: IFocusTrapZoneProps) {
     const prevForceFocusInsideTrap = prevProps.forceFocusInsideTrap !== undefined ? prevProps.forceFocusInsideTrap : true;
     const newForceFocusInsideTrap = this.props.forceFocusInsideTrap !== undefined ? this.props.forceFocusInsideTrap : true;
+    const prevDisabled = prevProps.disabled !== undefined ? prevProps.disabled : false;
+    const newDisabled = this.props.disabled !== undefined ? this.props.disabled : false;
 
-    if (!prevForceFocusInsideTrap && newForceFocusInsideTrap) {
+    if ((!prevForceFocusInsideTrap && newForceFocusInsideTrap) || (prevDisabled && !newDisabled)) {
```

Comment:
I feel like we should only bring focus into the zone if it currently has `forceFocusInsideTrap`. In the case where `forceFocusInsideTrap` is false we shouldn't move focus even if it had been disabled.

Reply:
Good catch! Thanks.

Reply:
Actually this gets called in `componentDidMount` regardless of `forceFocusInsideTrap`, and my testing has shown me why.  😆This will be needed to bring focus into FTZ once they are enabled, like they have just been mounted.